maintenance_plan/models/maintenance_equipment.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2017 Camptocamp SA
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from dateutil.relativedelta import relativedelta
+from odoo import _, api, fields, models
+
+
+def get_relativedelta(interval, step):
```

Comment:
As this is already defined in `maintenance_plan.py` you could just import it with `from .maintenance_plan import get_relativedelta`

Reply:
Thanks! Fixed.